include/sys/arc_impl.h

```diff
@@ -240,6 +240,7 @@ struct arc_buf_hdr {
 	arc_buf_contents_t	b_type;
 	arc_buf_hdr_t		*b_hash_next;
 	arc_flags_t		b_flags;
+	uint8_t			b_complevel;
```

Comment:
Adding this `uint8_t` here is going to throw off the packed size of this structure, increasing it by a full 64-bits.  I'd suggest moving it in to the existing hole after `b_type`, which will cost us nothing and explicitly reserving the remaining hole space for documentation purposes.  You can use `pahole(1)` to check the packed alignment.

```c
        arc_buf_contents_t      b_type;
        uint8_t                 b_complevel;
        uint8_t                 b_reserved1     /* reserved for future use */
        uint16_t                b_reserved2     /* reserved for future use */
        arc_buf_hdr_t           *b_hash_next;
        arc_flags_t             b_flags;
```

Reply:
okay. will do with next commit